### ISSUE TITLE
Support for Walk With Me: Do You Know Your Walking Routine?

### ISSUE THREAD

**Issue by InternalLoss** [sepalani/GamestatsHTTP#74]
We have discussed this in private before, but thought it'd be prudent to create an issue ticket for tracking.

AFAIR, Walk With Me expects there to always be *some* level of data available in the database before it's able to start uploading data. As we do not have any data, it fails, never uploading its data. I have a PAL copy and can probably setup a WEP AP with some pain if necessary, but @sepalani also has a PAL copy.

**Comment by sepalani** [comment]
Yes, technically I managed to grab 2 PAL copies _(one new, one used)_ from my trip in Paris. I'll also try to grab the japanese version at some point: https://buyee.jp/item/search/query/歩いてわかる+生活リズム
